Configuration/Flexforms/Calendar.xml

```diff
@@ -60,6 +60,16 @@
                             </config>
                         </TCEforms>
                     </settings.initialDocument>
+                    <settings.showEmptyYears>
+                      <TCEforms>
+                        <exclude>1</exclude>
+                        <label>LLL:EXT:dlf/Resources/Private/Language/locallang_be.xlf:plugins.calendar.flexform.showEmptyYears</label>
+                        <config>
+                          <type>check</type>
+                          <default>1</default>
```

Comment:
If possible, when introducing new options the default should always be the old behavior in order to make updates easier. So this should be `0`.

```suggestion
                          <default>0</default>
```